modules/exploits/multi/http/ubiquiti_unifi_log4shell.rb

```diff
@@ -150,6 +150,6 @@ def exploit
     wait_until { @search_received && (!handler_enabled? || session_created?) }
     handler
   ensure
-    cleanup
+    cleanup_service
```

Comment:
I'd guess this one isn't needed then either @adfoster-r7 - the exploit driver should clean it up. Although the one above in `check` may be needed... exploits' `check` runner doesn't seem to call cleanup:

https://github.com/rapid7/metasploit-framework/blob/c3465a8ad8369ef2eab3acd9d1a21b19feed4d3e/lib/msf/base/simple/exploit.rb#L212